unitTests/ui/DynamicFacet/DynamicRangeFacetTest.ts

```diff
@@ -0,0 +1,86 @@
+import * as Mock from '../../MockEnvironment';
+import { DynamicRangeFacet, IDynamicRangeFacetOptions, DynamicRangeFacetValueFormat } from '../../../src/ui/DynamicFacet/DynamicRangeFacet';
+import { DynamicRangeFacetTestUtils } from './DynamicRangeFacetTestUtils';
+import { FacetType } from '../../../src/rest/Facet/FacetRequest';
+
+export function DynamicRangeFacetTest() {
+  describe('DynamicRangeFacet', () => {
+    let test: Mock.IBasicComponentSetup<DynamicRangeFacet>;
+    let options: IDynamicRangeFacetOptions;
+
+    beforeEach(() => {
+      options = { field: '@field' };
+      initializeComponent();
+    });
```

Comment:
A lot of tests call `initializeComponent()` a second time after setting some options.

Should those with no special options be moved into a nested "with default/common options" `describe`, and this `initializeComponent` be moved into its `beforeEach`?

Reply:
Although it would make less repetition,  I don't want my UT so be too nested and I want their order to make sense e.g.:
`should extract the endInclusive correctly "Inclusive"`
`should extract the endInclusive correctly "Exclusive"`

one can be tested without overriding options while for the other I need to, but having the later in a separate nested describe would make it harder  to look at the tests because they would be separated